packages/api/routes/apikeys/key/getKeysByUserMiddleware.js

```diff
@@ -0,0 +1,17 @@
+const db_apikey = require("../../../lib/db.apikey");
+
+// Return a function for getting all API keys for a given user.
+module.exports = function getKeysByUser() {
+  return async (req, res, next) => {
+    const apikey = await db_apikey.attach();
+    const user = req.query.user ? req.query.user.trim() : req.query.user;
+    const doc = user
+      ? await apikey.getKeysByUser(user)
+      : {
+          error: "Invalid Parameter",
+          message: "Username Missing: Username cannot be empty. It is needed to identify and fetch API Keys."
```

Comment:
I suggest changing this to "user id".  The value for `user` can be any string, but it's meant to be the user's UUID.
```suggestion
          message: "user id missing: user id cannot be empty. It is needed to create and assign a new API Key."
```